slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/DbSlashingProtection.java

```diff
@@ -68,8 +70,16 @@ public DbSlashingProtection(
       final ValidatorsDao validatorsDao,
       final SignedBlocksDao signedBlocksDao,
       final SignedAttestationsDao signedAttestationsDao,
-      final MetadataDao metadataDao) {
-    this(jdbi, validatorsDao, signedBlocksDao, signedAttestationsDao, metadataDao, new HashMap<>());
+      final MetadataDao metadataDao,
+      final LowWatermarkDao lowWatermarkDao) {
+    this(
+        jdbi,
+        validatorsDao,
+        signedBlocksDao,
+        signedAttestationsDao,
+        metadataDao,
+        lowWatermarkDao,
+        new HashMap<>());
```

Comment:
if the map is not meant to be modified, you can pass `Collections.emptyMap` instead.

Reply:
it does get changed, so can't use an emptyMap.